samples/tools/autogenbench/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "autogenbench"
+authors = [
+  { name="Autogen Team", email="autogen@microsoft.com" },
```

Comment:
The email address we use for the AutoGen team is `auto-gen@outlook.com`. Should we use this one?

Reply:
Ok, will do.